crates/sui-core/src/consensus_adapter.rs

```diff
@@ -334,11 +429,18 @@ impl ConsensusAdapter {
             epoch_store.record_epoch_pending_certs_process_time_metric();
         }
 
+        // later populate this with the narwhal nodes that had lowest reputation scores
+        let low_scoring_authorities = vec![];
```

Comment:
Can we remove this for now to make this PR cleaner? When we introduce narwhal scores we can always add this parameter

Reply:
I have already made the commitment to including this functionality during the planning meeting

Reply:
This is not a question about commitment - let's keep PRs limited to one change at a time. There is no problem with adding low-score related things in the future PRs. Mixing in random pars of future PRs makes things more complicated to review and raise questions that are hard to answer without seeing the future PR(like for example, why is low_scoring_authorities defined here? Why not abstract all of this out of consensus adapter into some kind of `ExcludeAuthority` trait.